apps/emqx_authz/src/emqx_authz.erl

```diff
@@ -100,10 +100,10 @@ init() ->
     Sources = emqx_conf:get(?CONF_KEY_PATH, []),
     ok = check_dup_types(Sources),
     NSources = create_sources(Sources),
-    ok = emqx_hooks:add('client.authorize', {?MODULE, authorize, [NSources]}, -1).
```

Comment:
IMO, we should keep the `client.authorize` hook name metric name like `client.authenticate`

Renaming `client.authorization.*` to `authorization.*`  is enough?